microsoft-edge/progressive-web-apps-chromium/ux.md

```diff
@@ -0,0 +1,108 @@
+---
+description: This guide gives you an overview of the user experience of PWAs on Microsoft Edge and Windows.
+title: The user experience of PWAs
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 09/15/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: progressive web apps, PWA, Edge, Windows, desktop, installation, integration, microsoft store, ux
+---
+# The user experience of PWAs  
+
+On Windows, PWAs are first-class app citizens and any device running Microsoft Edge gets full access to the technologies and characteristics of Progressive Web Apps.  
+
+## Installing a PWA  
+
+When Microsoft Edge determines that a website is an installable PWA, the **App available** button is displayed in the address bar.  
+
+Select the **App available** button to install the PWA.  
+
+:::image type="complex" source="./media/edge-app-install-flyout.png" alt-text="The installation prompt in Microsoft Edge" lightbox="./media/edge-app-install-flyout.png":::
```

Comment:
This will need updating when the new UI lands.

Reply:
Thanks, making a note of it, to update screenshots in the new future.